src/scenes/home/team/team.js

```diff
@@ -1,58 +1,93 @@
-import BoardMembers from './boardMembers';
-import ExecutiveStaff from './executiveStaff';
 import styles from './team.css';
 
-const Team = () => (
-  <div>
-    <QuoteBanner
-      author="Abraham Lincoln"
-      quote="To care for him who shall have borne the battle and for his widow, and his orphan."
-    />
-
-    <Section title="Our Board" theme="white">
-      <div className={styles.boardMembers}>
-        {BoardMembers.map(boardMember => (
-          <TeamCard
-            key={`${boardMember.name} + ${boardMember.role}`}
-            name={boardMember.name}
-            role={boardMember.role}
-            description={boardMember.description}
-            imageSrc={boardMember.imageSrc}
-          />
-        ))}
-      </div>
-      <div className={styles.foundingMembers}>
-        <p>
-          Operation Code deeply appreciates the time, energy, and hard work of
-          our <b>Founding Board Members</b>: Aimee Knight, Laura Gomez, Pete
-          Runyon, Josh Carter, and Nick Frost.
-        </p>
-
-        <p style={{ textAlign: 'center' }}>
-          <em>Thank you for setting us up for success!</em>
-        </p>
-      </div>
-    </Section>
-
-    <Section title="Our Executive Staff" theme="white">
-      <div className={styles.executiveStaff}>
-        {ExecutiveStaff.map(staff => (
-          <TeamCard
-            key={`${staff.name} + ${staff.role}`}
-            name={staff.name}
-            role={staff.role}
-            slack={staff.slackUsername}
-            email={staff.email}
-            isBoard={false}
-          />
-        ))}
+class Team extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      teamMembers: null,
+      errorResponse: false
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get('https://api.operationcode.org/api/v1/team_members')
+      .then(response => this.setState({ teamMembers: response.data }))
+      .catch(() => this.setState({ errorResponse: true }));
+  }
+
+  render() {
+    return (
+      <div>
+        <QuoteBanner
+          author="Abraham Lincoln"
+          quote="To care for him who shall have borne the battle and for his widow, and his orphan."
+        />
+
+        {this.state.errorResponse && (
+          <p className={styles.teamError}>
+            Whoops! Something went wrong… please check back later.
+          </p>
+        )}
+
+        {this.state.teamMembers && (
+          <Section title="Our Board" theme="white">
+            <div className={styles.boardMembers}>
+              {this.state.teamMembers
+                .filter(board => board.group === 'board')
+                .map(members => (
+                  <TeamCard
+                    key={`${members.name} + ${members.role}`}
+                    name={members.name}
+                    role={members.role}
+                    description={members.description}
+                    imageSrc={members.image_src}
```

Comment:
Images can't be loaded right now. I think it is because of how they are stored in the database.